daprdocs/content/en/reference/components-reference/supported-middleware/middleware-bearer.md

```diff
@@ -25,6 +25,10 @@ spec:
       value: "<your token audience; i.e. the application's client ID>"
     - name: issuer
       value: "<your token issuer, e.g. 'https://accounts.google.com'>"
+    - name: pipelineType
```

Comment:
Having thought about this, it is okay to add these to each component, but need to say when these are used reading the component as a resource file.